interface/wx/statbox.h

```diff
@@ -97,5 +137,51 @@ class wxStaticBox : public wxControl
                 const wxPoint& pos = wxDefaultPosition,
                 const wxSize& size = wxDefaultSize, long style = 0,
                 const wxString& name = wxStaticBoxNameStr);
-};
 
+    /**
+        Creates the static box with the window as a label.
+
+        This method can only be called for an object created using its default
+        constructor.
+
+        See the constructor documentation for more details.
+
+        Currently this overload is only available in wxGTK, use
+        @c wxHAS_WINDOW_LABEL_IN_STATIC_BOX to check whether it can be used at
+        compile-time.
+
+        @since 3.1.1
+     */
+    wxStaticBox(wxWindow* parent, wxWindowID id,
```

Comment:
I guess this should be `Create()` instead of the constructor?
And its also available for `wxMSW`.

Reply:
Fixed, thanks!